baselines/ppo1/run_soccer.py

```diff
@@ -48,17 +52,33 @@ def main():
     boolean_flag(parser, 'save-model', default=True)
     boolean_flag(parser, 'load-model', default=False)
     parser.add_argument('--model-dir')
+    parser.add_argument('--timesteps_per_actorbatch', type=int, default=4096)
```

Comment:
As I said to you, I added PPO hypers as arguments to automatize random search - which is the best thing I did to improve reward in my experiments.